src/variation/translators/ambiguous_translator_base.py

```diff
@@ -78,22 +82,26 @@ async def get_grch38_data_ambiguous(
             AmbiguousType.AMBIGUOUS_2,
             AmbiguousType.AMBIGUOUS_5,
         }:
+            # `g_to_grch38` return inter-residue, but we want residue here
+            # so we increment start by 1
             grch38 = await self.mane_transcript.g_to_grch38(
-                ac, classification.pos1, classification.pos2
+                ac, classification.pos1 + 1, classification.pos2
             )
             if grch38:
-                pos1, pos2 = grch38["pos"]
-                new_ac = grch38["ac"]
+                pos1, pos2 = grch38.pos
+                new_ac = grch38.ac
         elif classification.ambiguous_type == AmbiguousType.AMBIGUOUS_7:
+            # `g_to_grch38` return inter-residue, but we want residue here
+            # so we increment start by 1
```

Comment:
I don't love this, but needed a quick fix. I think I may make an issue in Cool-Seq-Tool to provide start/end residue mode arguments.